lib/ecto/query.ex

```diff
@@ -436,6 +436,10 @@ defmodule Ecto.Query do
     Distinct.build(query, binding, expr, __CALLER__)
   end
 
+  defmacro distinct(query, expr) do
```

Comment:
We need to add docs. Even if it will be just "Adds a distinct clause. See `distinct/3`."